test/test_stream_reset.py

```diff
@@ -89,19 +89,22 @@ def test_reset_stream_keeps_flow_control_correct(self,
         c.clear_outbound_data_buffer()
 
         f = frame_factory.build_data_frame(
-            data=b'some data!',
+            data=b'some data',
```

Comment:
@Kriechi I don't follow this change, did the `!` have some effect on the test?

Reply:
Nothing important - some tests used `some data`, or `some data!`, or `foobar`.
I simply wanted to unify the test messages.